code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -314,6 +276,41 @@
 		return FALSE
 	return TRUE
 
+/proc/makeNewConstructFromClass(construct_class, theme, target, creator, cultoverride, loc_override)
+	switch(construct_class)
+		if("Juggernaut")
+			switch(theme)
+				if(THEME_CULT)
+					makeNewConstruct(/mob/living/simple_animal/hostile/construct/juggernaut, target, creator, cultoverride, loc_override)
```

Comment:
maybe have this be
```
if(THEME_CULT)
    if(ISCULTIST(creator))
        #make it a proper cult
    else
        #make it a non cultist construct 
````
and drop the idea of fake/real cult themes, since they're aesthetic things for the soulstone